.github/workflows/release-builder.yml

```
@@ -617,14 +627,16 @@ jobs:
       - name: ⚙️ Set up Node.js for Version Update
         uses: ./.github/actions/setup-node
         with:
-          node-version: 'lts/*'
+          node-version: ${{ inputs.node-version }}
+          cache: 'pnpm'
```

actionlint (1.7.12)

[error] input "cache" is not defined in action "Setup Node.js" defined at "./.github/actions/setup-node". available inputs are "dependency-path", "node-version", "package-manager"

(action)